docs/units-and-global-variables.rst

```diff
@@ -262,6 +262,15 @@ For more information, see the section on :ref:`address`.
     to make safe Ether transfers, always check the return value of ``send``, use ``transfer`` or even better:
     Use a pattern where the recipient withdraws the money.
 
+.. warning::
+    Due to the fact that the EVM considers a call to a non-existing contract to always succeed,
+    Solidity includes an extra check using the ``extcodesize`` opcode when performing external calls.
+    This ensures that the contract that is about to be called either actually exists (it contains code)
+    or an exception is raised.
+
+    The low-level ``.call()``, ``.delegatecall()`` and ``.staticcall()`` **do not** include this
```

Comment:
This is from #6751. I mostly just copied the clarification from #9847.

I need some help here though: do we perform this extra check on `.transfer()` and `.send()` too? They're a bit more high-level so I'm assuming we do but I'd have dig into the code to be sure.

Reply:
Also, are there any other places where I should include this warning?

Reply:
No, transfer and send are supposed to work on non-contracts. Essentially: Everything that works on `address` does not have the check (because it is just an address). Calls on contract types have the check.